### ISSUE TITLE
[UI] Cluster Template Issues - Read-Only access user sees Clone/Delete actions

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#21569]
- [ ] **_User shared template with Read-Only access sees Clone and Delete actions in UI:_**

1. Be shared template as read-only
2. View Cluster Templates page
3. Open function menu. You can see options for cloning and deleting.

RESULT: Ideally these should be hidden from read-only users. Read-only user should only be able to perform "View in API" action (This may be an API issue and needs separate GH issue?).

**Comment by davidnuzik** [comment]
Reopening per our discussion; this may need to be reverted. The API should be hiding these actions. Dynamic RBAC needed.

**Comment by davidnuzik** [comment]
Version: master-head (7/25/19)

Closing the issue. Confirmed we can see all actions again:
![image](https://user-images.githubusercontent.com/45179589/61890553-8762e200-aebc-11e9-8249-e3c33f0ad769.png)

Based on our discussion, the UI should not be doing things the API is failing to do. The API shows the user has all these actions at their disposal so the UI should not be trying to override things like this. We need Dynamic RBAC to fix this issue.

As such, we reverted the change. This is how things work everywhere else too and how it's been for a while. Down the road we may add in Dynamic RBAC later, although this would require significant effort.
